src/Cli/dotnet/commands/dotnet-workload/update/WorkloadUpdateCommand.cs

```diff
@@ -110,7 +110,7 @@ public override int Execute()
                             {
                                 return;
                             }
-                            UpdateWorkloads(false, manifestUpdates, offlineCache, context);
+                            UpdateWorkloadsWithInstallRecord(_sdkFeatureBand, manifestUpdates, false, context, offlineCache);
```

Comment:
I think this method doesn't exist after the refactoring, so you'll have to go back and deal with the conflicts.